src/main/scala/uk/gov/hmrc/SbtAutoBuildPlugin.scala

```diff
@@ -140,26 +141,27 @@ object HeaderSettings {
           .getOrElse(newText)
       }
     })
-  }
 
-  val commentStyles: Map[FileType, CommentStyle] = Map(
-    FileType.scala   -> retainYearCommentCreator(CommentStyle.cStyleBlockComment),
-    FileType.conf    -> retainYearCommentCreator(CommentStyle.hashLineComment),
-    FileType("html") -> retainYearCommentCreator(CommentStyle.twirlStyleBlockComment)
-  )
+  val commentStyles: Map[FileType, CommentStyle] =
+    Map(
+      FileType.scala   -> CommentStyle.cStyleBlockComment,
```

Comment:
strictly not necessary, `java` and `scala` already defined (and using `cStyleBlockComment`)